t/node/upstream.t

```diff
@@ -394,3 +394,119 @@ GET /t
 {"error_msg":"can not delete this upstream, route [1] is still using it now"}
 --- no_error_log
 [error]
+
+
+
+=== TEST 17: multi nodes with `node` mode to pass host
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/upstreams/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "nodes": {
+                        "localhost:1979": 1000,
+                        "127.0.0.1:1980": 1
```

Comment:
I am not very familiar with test grammar.
With weight 1000:1, the request should hit localhost:1979.
In the Test 18 below, should host be `localhost`?

Reply:
1979 is not reachable. It is used to test that we will use a new host for the retry.